templates/kubernetes/index.html

```diff
@@ -204,9 +204,10 @@ <h2>Kubernetes with Canonical support</h2>
           </li>
           <li class="p-list__item is-ticked">Fine-grained service placement</li>
         </ul>
-        <p>
-          <a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a>
-        </p>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="/kubernetes/install#multi-node">Install Charmed Kubernetes&nbsp;&rsaquo;</a></li>
+          <li class="p-inline-list__item"><a class="p-link--external" href="https://assets.ubuntu.com/v1/c5f59e1c-Enterprise_Kubernetes+Datasheet_16.09.21.pdf">Read the datasheet</a></li>
```

Comment:
I just got to know that it doesn't require an external link if it is for something on pages.ubuntu.com or assets.ubuntu.com, like ebooks, whitepapers, pdfs, etc. so it doesn't seem to be required the `p-link--external` class here.

```suggestion
          <li class="p-inline-list__item"><a href="https://assets.ubuntu.com/v1/c5f59e1c-Enterprise_Kubernetes+Datasheet_16.09.21.pdf">Read the datasheet&nbsp;&rsaquo;</a></li>
```